components/atoms/Textarea/text-area.tsx

```diff
@@ -1,38 +1,64 @@
-import React, { ChangeEvent, useRef } from "react";
+import React, { ChangeEvent, useEffect, useRef } from "react";
 import clsx from "clsx";
 
 interface TextareaProps extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {
   defaultRow?: number;
   onChangeText?: (value: string) => void;
+  typewrite?: boolean;
```

Comment:
I think this should be a separate `typewriter-textarea` atom or `molecule` and the `text-area` atom should remain as is. We only need this typewriter functionality in one place at the moment